post-receive-hook.sh

```diff
@@ -0,0 +1,11 @@
+#!/bin/sh
+set -e
+while read oldrev newrev refname
+do
+  if [ "$refname" = "refs/heads/master" ]
```

Comment:
Would it possibly make more sense for the publish branch to be named `gw-pages`?  That would complete the analogy with github pages, and it would make it a bit more difficult to accidentally publish something that you mean to keep private.

Reply:
Hm... if the ref-to-publish is named `gw-pages`, then maybe we don't need a "publish" button at all.